x-pack/plugins/alerting/server/data/r_rule/types/r_rule_attributes.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { WeekdayStr } from '@kbn/rrule';
+
+type RRuleFreq = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface RRuleAttributes {
```

Comment:
are we not sharing this type with the rule type for snooze

Reply:
we are, hence this is in its own data type folder `r_rule`